src/styles/index.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import 'globals';
+@import 'utils/variables';
```

Comment:
Ensure that the file `utils/variables.scss` exists and contains the necessary variable definitions used in other SCSS files, such as `$primary-color-text-card`, `$secondary-color-text-card`, and `$primary-button-color`.